frontend/src/lib/components/authentication/LoginPage.svelte

```diff
@@ -73,6 +73,8 @@ Candidate login page. This component also takes care of the login process.
           {/if}
           <button type="submit" class="btn btn-primary mb-md w-full max-w-md"
```

Comment:
Should use the new button component

Reply:
I think it's better to keep to the old style and not use the component yet, in order to not mix styles. This file should be refactored later to use the new components.